src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -923,6 +987,9 @@ public override void ErrorHandler(object sender, BuildErrorEventArgs e)
             // Keep track of the number of error events raised 
             errorCount++;
 
+            // Determine the mapping of properties to output
+            PropertyMapping(e);
```

Comment:
If you change the name of the method, it can be self-documenting without the comment:

```suggestion
            PopulatePropertyMap(e);
```